examples/samplecontainer/README.md

```diff
@@ -0,0 +1,18 @@
+To build, from this directory, run:
+
+```sh
+bazel build //cmd/clusterregistry //examples/slackcontroller
+cp "$(bazel info bazel-bin)/cmd/clusterregistry/clusterregistry" \
+  "$(bazel info bazel-bin)/examples/slackcontroller/slackcontroller" \
+  ./contents
```

Comment:
Do you need .gitingore entries for these binaries?

Reply:
That's a good point. Done.